client/lcd/swagger-ui/swagger.yaml

```diff
@@ -521,9 +556,629 @@ paths:
         200:
           description: Account information on the blockchain
           schema:
-            $ref: "#/definitions/AccountQueryResponse"
+            type: object
+            properties:
+              type:
+                type: string
+              value:
+                type: object
+                properties:
+                  account_number:
+                    type: string
+                  address:
+                    type: string
+                  coins:
+                    type: array
+                    items:
+                      $ref: "#/definitions/Coin"
+                  public_key:
+                    type: string
+                  sequence:
+                    type: string
         404:
```

Comment:
Did we choose to make this a 404? We don't know if someone holds that address but hasn't don't any transaction yet. Querying such an account would result in an error. I propose we do either serve a 400 or serve a default response with an empty account and a 200.

Reply:
I rechecked the code. If there is no content about this account address, 204 error code will be returned.
```
if len(res) == 0 {
   w.WriteHeader(http.StatusNoContent)
   return
}
```